docs/content/modeling/user-groups.mdx

```diff
@@ -21,11 +21,11 @@ import {
 
 <DocumentationNotice />
 
-In this guide you will learn how to add users to groups and grant groups access to an <ProductConcept section="what-is-an-object" linkName="object" /> using <ProductName format={ProductNameFormat.ProductLink}/>.
+This guide explains how to add users to groups and grant groups access to an <ProductConcept section="what-is-an-object" linkName="object" /> using <ProductName format={ProductNameFormat.ProductLink}/>.
```

Comment:
Try not to start guides with an intro like this, it's not really necessary. You can jump right in with, 
To add users...

```suggestion
To add users to groups and grant groups access to an <ProductConcept section="what-is-an-object" linkName="object" /> using <ProductName format={ProductNameFormat.ProductLink}/>.
```